### PR TITLE
Fixes a typo in the name of the 3d oranges

### PR DESCRIPTION
**Pull request by Lett1**
Just a lil typo fix.

### CODE REVIEW

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -178,7 +178,7 @@
 
 /obj/item/reagent_containers/food/snacks/grown/citrus/orange_3d
 	seed = /obj/item/seeds/orange
-	name = "extradminesional orange"
+	name = "extradimensional orange"
```
Comment (Jalleo) [inline code/modules/hydroponics/grown/citrus.dm]: It might lf been done on purpose

**Review by SpaceManiac — APPROVED** [review]

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (Lett1) [reply code/modules/hydroponics/grown/citrus.dm]: Whats wrong with it now?

Comment (nemvar) [reply code/modules/hydroponics/grown/citrus.dm]: extrADMINesional

Comment (Lett1) [reply code/modules/hydroponics/grown/citrus.dm]: Oh haha

**Pull request merged**